ocean_provider/routes/compute.py

```diff
@@ -82,13 +89,13 @@ def computeDelete():
         response = requests_session.delete(
             get_compute_endpoint(),
             params=body,
-            headers={"content-type": "application/json"},
+            headers=standard_headers,
```

Comment:
this looks fishy. in the future, we will have support for more endpoint/protocol, I prefer to keep this customizable for each.
2nd, I'm not sure if it will impact build_down_reponse (which we don't want to)